docs/usage.md

```diff
@@ -81,16 +80,15 @@ Steps to incorporate the recommendations
 
 
 # Note 
-* We recommend using the first method to update configuration changes.
-* For HA setups apply the recommended configuration to all the system and then restart the services starting with primary
-* If you have changed some parameters in the `/var/lib/pgsql/16/data/postgresql.conf` file and the same parameter is present in the `/var/lib/pgsql/16/data/postgresql.auto.conf` file, the parameter in `/var/lib/pgsql/16/data/postgresql.conf` will be overridden by the one in `/var/lib/pgsql/16/data/postgresql.auto.conf`. Therefore, we recommend using the first method to edit system resources.
+* For HA setups apply the recommended configuration to all the systems. Restart the services on primary node first then secondary nodes.
```

Comment:
how check which is the primary node, can give prod doc reference here for easy understanding

Reply:
this note should be present before apply steps. and please use md format for highlight "notes". Please check other sP docs for a reference